### ISSUE TITLE
add address search

### ISSUE THREAD

**Issue by polly64** [colouring-cities/colouring-london#253]
@matkoniecz would be good to add this as part of CLPV tool as people will also want to search for planning application status for a specific building,  as well as it being  generally useful

**Comment by polly64** [comment]
@matkoniecz for discussion

**Comment by matkoniecz** [comment]
First question is data source. We can use OS for UK if we can and populate our primitive search table (or from OpenStreetMap? Other source?)

Or run proper geolocation server like Nominatim but it will take some effort.

**Comment by polly64** [comment]
@is nominatim open and comprehensive? If so think worth it as addresses so important . OS data not available

**Comment by matkoniecz** [comment]
@polly64 Nominatim is software used for geocoding based on (primarily) OpenStreetMap data. See https://nominatim.org/

We may qualify for https://operations.osmfoundation.org/policies/nominatim/ 

> No heavy uses (an absolute maximum of 1 request per second).
> Clearly display [attribution](https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines) as suitable for your medium.

where we would need to integrate external service without running our own.

**Comment by polly64** [comment]
@matkoniecz @mdsimpson42 what do you think shall we try this? maybe for discussion at next meeting?

**Comment by matkoniecz** [comment]
https://github.com/colouring-cities/colouring-britain/issues/65#issuecomment-1885961677 may be also potentially useful

**Comment by matkoniecz** [comment]
It is nice to have, the question is whether we should work on it right now or is other issue more important.

Note, adding search by city/town/place name is much easier and could be done first.

**Comment by polly64** [comment]
@matkoniecz yes if complicated leave and we'll do after April and would be great do city/town place name if easier and quick

**Comment by matkoniecz** [comment]
city/town place name can be basically added as soon as source of names + locations is decided on/found/available
